### PR TITLE
dex ag [v5]

### PR DESCRIPTION
**Pull request by SteveMieskoski**
### Devop

- [x] Updated CHANGELOG.md
- [x] Add PR label
- [x] No hard-coded strings

### CODE REVIEW

File: src/layouts/InterfaceLayout/containers/SwapContainer/SwapContainer.vue [hunk 1]
```diff
@@ -880,6 +903,7 @@ export default {
       }
     },
     openConfirmModal(swapDetails) {
+      console.log(swapDetails, 'WAAAAAAAAAT');
```
Comment (gesseekur) [inline src/layouts/InterfaceLayout/containers/SwapContainer/SwapContainer.vue]: wat

File: compileBaseSwapOptions.js [hunk 2]
```diff
@@ -144,6 +143,40 @@ class CompileSwapOptions {
     }
   }
 
+  async getDexAgSupported(    priorCollected = {
+    ETH: {},
+    other: {}
+  }) {
+    try {
+      const tokenList = await this.get(
+        'https://api-v2.dex.ag/token-list-full'
```
Comment (kvhnuke) [inline compileBaseSwapOptions.js:152]: use mewapi link
`https://dexag.mewapi.io/token-list-full`
`dexag.mewapi.io == api-v2.dex.ag`

File: src/layouts/InterfaceLayout/containers/NFTManagerContainer/NFTManagerContainer.vue [hunk 3]
```diff
@@ -285,6 +285,7 @@ export default {
       return null;
     },
     activeAddress() {
+      // return "0x51e723b14b160b9b420cbf7ccdb889f36213f0f3"
```
Comment (kvhnuke) [inline src/layouts/InterfaceLayout/containers/NFTManagerContainer/NFTManagerContainer.vue:288]: commented code

File: src/layouts/InterfaceLayout/containers/SwapContainer/components/ProvidersRadioSelector/ProviderInfoList/ProviderInfoList.vue [hunk 4]
```diff
@@ -46,7 +47,8 @@ export default {
         kybernetwork: KyberNetwork,
```
Comment (kvhnuke) [inline src/layouts/InterfaceLayout/containers/SwapContainer/components/ProvidersRadioSelector/ProviderInfoList/ProviderInfoList.vue:47]: why is kyber network still there?

File: src/layouts/InterfaceLayout/containers/SwapContainer/components/ProvidersRadioSelector/ProvidersRadioSelector.vue [hunk 5]
```diff
@@ -65,7 +67,9 @@
             <span class="slippage-text">{{
               otherTextDisplay(provider.additional.display)
             }}</span>
+            <!--            <span class="slippage-text"> {{ provider.additional.source }}</span>-->
```
Comment (kvhnuke) [inline src/layouts/InterfaceLayout/containers/SwapContainer/components/ProvidersRadioSelector/ProvidersRadioSelector.vue:70]: commented code

File: src/partners/partnersConfig/config.js [hunk 6]
```diff
@@ -13,7 +13,8 @@ const swapApiEndpoints = {
   base: 'https://swap.mewapi.io',
   changelly: '/changelly',
   bity: '/bity',
-  kyber: '/kyber'
+  kyber: '/kyber',
```
Comment (kvhnuke) [inline src/partners/partnersConfig/config.js:16]: kyber still there?

File: vueConfig/connections.js [hunk 7]
```diff
@@ -60,5 +60,8 @@ module.exports = [
   'ws://127.0.0.1:8178',
   'https://viewm.moonicorn.network:443',
   'https://localhost:1991', // server for bcvault
-  'https://rpc.atheios.org:443'
+  'https://rpc.atheios.org:443',
+  'wss://bridge.walletconnect.org:443',
+  'api.dex.ag:443',
+  'api-v2.dex.ag:443'
```
Comment (kvhnuke) [inline vueConfig/connections.js:66]: this should be just `dexag.mewapi.io:443` there is no reason for api v1 I believe?

File: vueConfig/mew.js [hunk 8]
```diff
@@ -12,7 +12,10 @@ const webpackConfig = {
     host: 'localhost',
     hotOnly: true,
     port: 8080,
-    headers: defaultConfigs.headers
+    headers: defaultConfigs.headers,
+    watchOptions: {
+      poll: true
+    }
```
Comment (kvhnuke) [inline vueConfig/mew.js:18]: I dont think this is necessary, standard watch should work

**Pull request merged**